Exygy/askdarcel-web
# Listing table

# Pull Request Template

## Issue Overview

## Description

This PR refactors uses of a global `<table>` element and leans more on DataTable component. It also pulls css out of global stylesheets and into module scss files. 

## How Can This Be Tested/Reviewed?

Visit any /service/SRVID page and check that the tables are uniform (except the table in Accordion which also has its own css pulled into Accordion.module.scss)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
